ctdcal/libODF_fit_ctd.py

```diff
@@ -34,75 +34,87 @@ def offset(offset, inArr):
     return inArr + offset
 
 
-def IESRho(s, t, p):
-    bars  = p * 0.1
+def IESRho(df,sal_col='CTDSAL',t_col='CTDTMP1',p_col='CTDPRS'):
```

Comment:
Don't need to edit this. This should be imported from gsw and extended if need be.